app/src/pages/outside/loginPage/pageBlocks/loginBlock/loginForm/loginForm.jsx

```diff
@@ -105,6 +113,10 @@ export class LoginForm extends React.Component {
     if (prevProps.lastFailedLoginTime !== this.props.lastFailedLoginTime) {
       this.blockLoginForm();
     }
+    const { badCredentials } = this.props;
+    if (badCredentials) {
```

Comment:
I think it needs a better condition check because now it fires when I type values in different fields, so I see validation errors even when I haven't submitted new values.
Maybe `if (badCredentials  !== prevProps.badCredentials && badCredentials) {`